api/nakadi-event-bus-api.yaml

```diff
@@ -1647,6 +1695,18 @@ definitions:
         type: string
         description: |
           Offset of the event being pointed to.
+      shift:
```

Comment:
@v-stepanov maybe we should actually make it be costly so that people won't replace new-offset by "BEGIN" + "old-offset" as shift.